webrtc-c/canary/src/Cloudwatch.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+namespace Canary {
+
+class Synchronization {
```

Comment:
Will this be only used here? If only used in CW logs then should it be encapsulated within that class as private?

Reply:
When I created it, it was meant for logs and monitoring. But, it doesn't apply to monitoring anymore. 

Yes, this is a good idea.